src/atomate2/forcefields/schemas.py

```diff
@@ -91,6 +93,7 @@ class ForceFieldTaskDocument(AseStructureTaskDoc, ForceFieldMeta):
     def from_ase_compatible_result(
         cls,
         ase_calculator_name: str,
+        calculator_meta: MLFF | dict,
```

Comment:
I think this is the only point where the interface is changing. Does it make sense to put a default `None` here to avoid complications for other users?

Reply:
I have made `calculator_meta` parameter in `ForceFieldTaskDocument.from_ase_compatible_result()` optional, 3ae94d4. Now, the `calculator_meta` is inferred from `ase_calculator_name` if it is `MLFF`.
Although the implementation logic gets a bit more complicated, is it acceptable?

Reply:
I prefer this version. Thanks!